chrome_manifest_v3/app.js

```
@@ -617,339 +606,330 @@ Gmail2Trello.App.prototype.markdownify = function (
-        self.lastSettingsSave = settings_string_k;
-    }
+  let self = this;
+  const setID = this.CHROME_SETTINGS_ID;
+  let settings = $.extend({}, this.popupView.data.settings);
+
+  // Delete large, potentially needing secure, data bits:
+  settings.description = '';
+  settings.title = '';
+  settings.attachments = [];
+  settings.images = [];
+  delete settings.description;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)